retriever/lib/install.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+
+from retriever.engines import choose_engine
+from retriever.lib.defaults import DATA_DIR
+from retriever.lib.scripts import SCRIPT_LIST
+from retriever.lib.tools import name_matches
+
+
+def _install(args, use_cache, debug, compile):
+    """Install scripts for retriever."""
+    engine = choose_engine(args)
+    engine.use_cache = use_cache
+
+    script_list = SCRIPT_LIST(force_compile=True)
+    scripts = name_matches(script_list, args['dataset'])
+    if scripts:
+        for script in scripts:
+            print("=> Installing", script.name)
```

Comment:
The quiet parameter goes into the `Script.download` function. The install statement is just to notify the user that something is happening. What is your call should we print or not?

Reply:
I don't have strong feelings on this. My thought was that since `quiet` is basically suppressing output most users would want to suppress all of it. @henrykironde - any thoughts?